modules/docs/arrow-docs/docs/docs/arrow/core/either/README.md

```diff
@@ -317,6 +317,21 @@ Example:
 null.rightIfNotNull { "left" }
 ```
 
+The inverse of `rightIfNotNull`, `rightIfNull`.
+If the value is null it will be transformed to the specified `Either.right` and the type will be `Nothing?`.
+If the value is not null than it will be transformed to the specified `Either.Left`.
+
+Example:
+
+```kotlin:ank
+// Results in Either.Left
```

Comment:
These aren't necessary, ank will display the result inlined below this snippet

Reply:
Thanks! should be corrected now :)